test/livebook_web/live/home_live_test.exs

```diff
@@ -231,6 +231,39 @@ defmodule LivebookWeb.HomeLiveTest do
     end
   end
 
+  describe "hubs sidebar" do
+    test "doesn't show with disabled feature flag", %{conn: conn} do
+      Application.put_env(:livebook, :feature_flags, hub: false)
+      {:ok, _view, html} = live(conn, "/")
+      Application.put_env(:livebook, :feature_flags, hub: true)
+
+      refute html =~ "HUBS"
+    end
+
+    test "render section", %{conn: conn} do
+      {:ok, _view, html} = live(conn, "/")
+      assert html =~ "HUBS"
+      assert html =~ "Add Hub"
+    end
+
+    test "render persisted hubs", %{conn: conn} do
+      hub = %Livebook.Hubs.Hub{
+        id: "foo",
+        name: "Foo",
+        token: "foo",
+        color: "#FF00FF"
+      }
+
+      assert Livebook.Hubs.save_hub(hub)
```

Comment:
```suggestion
      Livebook.Hubs.save_hub(hub)
```

Reply:
Perhaps it is worth creating a support module called `Fixtures`, where we have `create_hub(id, attrs \\ %{})`. You can use the Fixtures module that Phoenix would generate under `mix phx.gen.html` as a starting point!